src/components/Banner.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import styled from 'styled-components';
+import { graphql, StaticQuery } from 'gatsby';
+import { Card, Image } from 'semantic-ui-react';
+
+class Banner extends React.Component {
+
+  renderLogos = allLogos => (
+    <Card fluid >
+      <Card.Content className="card">
+        {this.getRandom(allLogos, 4).map(logo => {
+        if(logo.node.image.file.url != null){
```

Comment:
Does this actually happen? I'd think a logo would only be in the array if it has a url

Reply:
True, now that I think of it. When creating a logo, the imagefield is required.